test/test_cuda_mapper.cc

```diff
@@ -886,6 +886,78 @@ def fun(float(N, K) I) -> (O) {
   EXPECT_TRUE(code.find(kCUBReductionName) != std::string::npos);
 }
 
+/*
+ * Check that a 2D mean with fixed parameters mapped to 512 threads
+ * produce a reduction library call.
+ * In practice, check that the library call appears in the code.
+ */
+TEST_F(PolyhedralMapperTest, DISABLED_Mean2DNonParametric_512threads) {
```

Comment:
This test looks incorrect.
The reduction band is the result of tiling by 32, so it can only be mapped
to 32 threads, while the number of threads in the X direction is set to 512.